### ISSUE TITLE
AAT, I'd like a webview test that asserts HTML markup should not be visible in section titles

### ISSUE THREAD

**Issue by m1yag1** [openstax/cnx-automation#146]
# CASE NUM: C167408

## User story

```
AAT, I'd like a webview test that asserts HTML markup should not be visible in section titles
```

## Preconditions

```

```

## Steps

```
Look at any title of any section.
```

## Expected Result

```
It should read something like `1.4 The Language of Physics`

NOT

`<span class="os number">1.4</span><span class="os text">The Language of Physics</span>`
```

**Comment by Dantemss** [comment]
https://github.com/openstax/cnx-automation/pull/175

**Comment by m1yag1** [comment]
Resolved by #175
